source/MRViewer/ImGuiHelpers.h

```diff
@@ -139,6 +139,13 @@ MRVIEWER_API void PlotCustomHistogram( const char* str_id,
 
 /// begin typical state plugin window
 MRVIEWER_API bool BeginStatePlugin( const char* label, bool* open, float width );
+
+/// begin state plugin window with custom style.  If you use this function, you must call EndCustomStatePlugin to close the plugin correctly.
+MRVIEWER_API bool BeginCustomStatePlugin( const char* label, bool* open, bool* collapsed, float width, float menuScaling, float height = 0.0f, 
+    ImGuiWindowFlags flags = ImGuiWindowFlags_NoResize | ImGuiWindowFlags_AlwaysAutoResize | ImGuiWindowFlags_NoScrollbar | ImGuiWindowFlags_NoScrollWithMouse );
```

Comment:
please write comment about flags, and remove force flags from default value
also can we have ImVec2 size instead of float width and height (or at least it would be nice if this args are close in api)